ct/pocketbase.sh

```diff
@@ -52,6 +52,29 @@ function default_settings() {
   echo_default
 }
 
+function update_script() {
+  header_info
+  check_container_storage
+  check_container_resources
+  if [[ ! -f /etc/systemd/system/pocketbase.service || ! -x /opt/pocketbase/pocketbase ]]; then msg_error "No ${APP} Installation Found!"; exit; fi
+msg_info "Stopping ${APP}"
```

Comment:
fix indendation

Reply:
And remove Restarting section

Reply:
and test that it still works

Reply:
Sorry, looks like something didn't quite work with the code suggestions.

Reply:
Fixed. Tested the script on my pocketbase install.
Seems to work, but I am already on the latest version

![image](https://github.com/user-attachments/assets/3536798d-29c2-4ca5-9e86-586a87aa9184)